testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergHiveViewsCompatibility.java

```diff
@@ -117,6 +117,66 @@ public void testIcebergHiveViewsCompatibility()
         }
     }
 
+    @Test(groups = {ICEBERG, STORAGE_FORMATS, HMS_ONLY})
+    public void testIcebergMaterializedViewHiveViewsCompatibility()
+    {
+        try {
+            cleanupMaterializedViews();
+
+            List<QueryAssert.Row> hivePreexistingTables = onTrino().executeQuery("SHOW TABLES FROM hive.default").rows().stream()
+                    .map(list -> row(list.toArray()))
+                    .collect(toList());
+            List<QueryAssert.Row> icebergPreexistingTables = onTrino().executeQuery("SHOW TABLES FROM iceberg.default").rows().stream()
+                    .map(list -> row(list.toArray()))
+                    .collect(toList());
+
+            //create base tables
+            onTrino().executeQuery("CREATE TABLE hive.default.hive_table AS SELECT 1 bee");
+            onTrino().executeQuery("CREATE TABLE iceberg.default.iceberg_table AS SELECT 2 snow");
+
+            //create views
+            onTrino().executeQuery("CREATE VIEW hive.default.hive_view AS SELECT * FROM hive.default.hive_table");
+            onTrino().executeQuery("CREATE VIEW iceberg.default.iceberg_view AS SELECT * FROM iceberg.default.iceberg_table");
```

Comment:
I suppose that the test purpose is materialized views and Trino views are already covered by `testIcebergHiveViewsCompatibility`. Can we remove these `CREATE VIEW` statements and focus on materialized views in this test?

Reply:
In this case, we need to assert that views can be accessed and decoded and that materialized views are filtered out.

Reply:
Ive simplified the test though to reduce replicating too much of the above test